<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>883</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #F0F2F5</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>160</y>
      <width>461</width>
      <height>401</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
background-color: #FFFFFF</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>401</width>
       <height>61</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    border: 1px solid rgb(212, 212, 212);
    border-radius: 10px;
    color: gray;
    padding-left: 15px;
}

QLineEdit:focus {
    color: black;
}</string>
     </property>
     <property name="dragEnabled">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string>Username</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>130</y>
       <width>401</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    border: 1px solid rgb(212, 212, 212);
    border-radius: 10px;
    color: gray;
    padding-left: 15px;
}

QLineEdit:focus {
    color: black;
}</string>
     </property>
     <property name="placeholderText">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonLogin">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>250</y>
       <width>401</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>VNI-Helve-Condense</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: #0866FF;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: #0866FF;
}

QPushButton:hover {
    background-color: #038EFF;
    border-color: #038EFF;
}

</string>
     </property>
     <property name="text">
      <string>Log in</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>-20</y>
      <width>331</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="images.qrc">:/fblogo.svg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>90</y>
      <width>361</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Facebook helps you connect and share with the people in your life.</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>883</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButtonLogin</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_button_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>580</x>
     <y>436</y>
    </hint>
    <hint type="destinationlabel">
     <x>235</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_button_clicked()</slot>
  <slot>FadedText()</slot>
 </slots>
</ui>
